.yamato/build_win.yml

```diff
@@ -0,0 +1,19 @@
+name: build_win
+
+agent:
+  type: Unity::VM
+  image: platform-foundation/windows-mono-bokken:latest
+  flavor: b1.xlarge
+
+interpreter: powershell
+
+commands:
+  - git submodule update --init --recursive
```

Comment:
Might be nice to have windows like other platforms where all logic is in a script, and the yml calls a single command. I think this also helps capture all output.

Reply:
Updated